From 51e96fd2c3312738d1254dff1d76e91a5abf8526 Mon Sep 17 00:00:00 2001
From: Antonio Rojas <arojas@archlinux.org>
Date: Sat, 24 Apr 2021 10:43:47 +0000
Subject: [PATCH] Fix build with OpenEXR 3

Add a header that is no longer transitively included

Part-of: <https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/-/merge_requests/2195>
---
 ext/openexr/gstopenexrdec.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ext/openexr/gstopenexrdec.cpp b/ext/openexr/gstopenexrdec.cpp
index eeb316cb18..32b1d03c38 100644
--- a/ext/openexr/gstopenexrdec.cpp
+++ b/ext/openexr/gstopenexrdec.cpp
@@ -29,6 +29,7 @@
 
 #include <ImfRgbaFile.h>
 #include <ImfIO.h>
+#include <ImfInt64.h>
 using namespace Imf;
 using namespace Imath;
 
